community/kernel/src/main/java/org/neo4j/kernel/impl/store/kvstore/RotationState.java

```diff
@@ -65,9 +69,16 @@ String stateName()
         {
             if ( !force )
             {
+                long rotationStart = Clock.SYSTEM_CLOCK.currentTimeMillis();
```

Comment:
Accessing `SYSTEM_CLOCK` directly is no different from calling `System.currentTimeMillis()` directly. The Clock interface is meant to be passed in, so it can be mocked in tests.